### PR TITLE
What's new in DevTools (115)

### PR DESCRIPTION
**Pull request by sofiayem**
WNDT 115

### CODE REVIEW

File: site/en/_partials/devtools/whats-new.md [hunk 1]
```diff
@@ -4,6 +4,22 @@ A list of everything that has been covered in the [What's New In DevTools](/tags
 
 {# $content #}
 
+### Chrome 115 {: #chrome115 }
+
+* [Elements improvements](/blog/new-in-devtools-115/#elements)
+* [New CSS subgrid badge](/blog/new-in-devtools-115/#subgrid)
```
Comment (rachelandrew) [inline site/en/_partials/devtools/whats-new.md:10]: Can we remove this section? Subgrid actually isn't in Chrome yet so it's a bit odd to announce the DevTools functionality. Hang onto the copy and we'll use it when subgrid ships.

File: site/en/blog/new-in-devtools-115/index.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+{% YouTube id='CrSmjooOEiE' %}
+
+<!-- $contentStart -->
+
+## Elements improvements {: #elements }
+
+### New CSS subgrid badge {: #subgrid }
+
+The **Elements** panel gets a new `subgrid` badge for [nested grids](https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout/Subgrid).
```
Comment (rachelandrew) [inline site/en/blog/new-in-devtools-115/index.md:29]:
```suggestion
The **Elements** panel gets a new `subgrid` badge for [subgrid](https://developer.mozilla.org/docs/Web/CSS/CSS_Grid_Layout/Subgrid). This feature is currently experimental in Chrome Canary.
```

File: site/en/blog/new-in-devtools-115/index.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+To inspect and debug a nested grid, click the badge. It toggles an overlay that shows columns, rows, and their numbers on top of the element in the viewport.
```
Comment (rachelandrew) [inline site/en/blog/new-in-devtools-115/index.md:31]:
```suggestion
To inspect and debug a nested grid that is a subgrid, and therefore inheriting the number and size of tracks from its parent, click the badge. It toggles an overlay that shows columns, rows, and their numbers on top of the element in the viewport.
```

**Review by rachelandrew — CHANGES_REQUESTED** [review]
I've corrected the definition of subgrid, subgrid are nested grids but only those where either grid-template-columns or grid-template-rows have a value of subgrid. Regular nested grids aren't subgrids, and won't get this badge. Also made it clear the feature is experimental and not in the current release.

**Review by rachelandrew — APPROVED** [review]

**Review by jecfish — APPROVED** [review]
Added review comments on docs.

**Pull request merged**